src/transformers/hoistLists.nix

```
@@ -0,0 +1,71 @@
+{ lib }: from: to:
+
+# Example from / to
+# - Lifting `imports` from: _imports, to: imports
+# - Lifting `options` from: _api, to: options
+#
+# Note: underscore used as mere convention to
+#       signalling to the user the  "private"
+#       nature, they won't be part of the final
+#       view presented to the user
+
+let
+  inherit (lib)
+    catAttrs
+    flip
+    pipe
+    mapAttrs
+    attrValues
+    foldl'
+    flatten
+    ;
+
+  /*mergeAttrsButConcatOn: merge attributes shallowly, but
+    concat values of a specific key into a list in that key
+
+    Type:
+      (key :: String -> AttrSet -> AttrSet) -> { ${key} :: List; ... }
+
+    */
+  mergeAttrsButConcatOn = key: x: y:
+    x // y // {
+      ${key} = flatten (catAttrs key [ x y ]);
```

`concatLists` should be enough here

```suggestion
      ${key} = concatLists (catAttrs key [ x y ]);
```